spikeinterface/sortingcomponents/peak_localization.py

```diff
@@ -32,7 +34,7 @@ def localize_peaks(recording, peaks, method='center_of_mass',
     peak_locations: np.array
         Array with estimated x-y location for each spike
     """
-    assert method in ('center_of_mass',)
+    assert method in ('center_of_mass', 'lsq_optimize')
```

Comment:
I would call this `monopolar`, since it assumes a monopolar current source (it's not really the optimization method that matters...)

Reply:
yes your are right

Reply:
What about "monopolar_triangularisation" ?